third-party/CMakeLists.txt

```diff
@@ -65,17 +65,17 @@ set(LUAROCKS_URL https://github.com/keplerproject/luarocks/archive/0587afbb5fe8c
 set(LUAROCKS_SHA1 61a894fd5d61987bf7e7f9c3e0c5de16ba4b68c4)
 set(LUAROCKS_MD5 0f53f42909fbcd2c88be303e8f970516)
 
-set(LIBUNIBILIUM_URL https://github.com/neovim/unibilium/archive/neovim.tar.gz)
-set(LIBUNIBILIUM_SHA1 ab22e465150458a4ea91b72de997a35f25df1bcf)
-set(LIBUNIBILIUM_MD5 b0749651429aa4c619baf44eedcd15c6)
+set(LIBUNIBILIUM_URL https://github.com/mauke/unibilium/archive/v1.1.1.tar.gz)
```

Comment:
I think your PR (mauke/unibilium#5) is not yet in this version; might be better to just use https://github.com/mauke/unibilium/commit/520abbc8b26910e2580619f669b5cc2c4ef7f864.